app/src/main/java/com/arctouch/codechallenge/api/TmdbApi.java

```diff
@@ -16,24 +27,70 @@
     String DEFAULT_LANGUAGE = "pt-BR";
     String DEFAULT_REGION = "BR";
 
+    int TIMEOUT = 60; //Seconds
+    int NUMBER_OF_THREADS = 20;
```

Comment:
How would you store these constants in an ideal scenario?

Reply:
These values could be stored in the build.gradle file (app), in which they would be a buildConfigField and could be accessed like BuildConfig.TIMEOUT for example. Also, there could be different timeouts for debug and release versions, for example, so on the debug flavor there would be a buildConfigField and on release another one, so on runtime that would be chosen correctly by gradle